modules/component/src/core/utils/view-engine-checks.ts

```diff
@@ -0,0 +1,33 @@
+import { getGlobalThis } from './get-global-this';
+
+/**
+ * envRunsIvy
```

Comment:
This can be left out (it's enough to have a description)? 
Or should it say more?

```suggestion

```